atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -146,6 +151,15 @@ public synchronized void runInBackground() {
         daemon.setDaemon(true);
         daemon.setName("BackgroundSweeper");
         daemon.start();
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            log.info("Shutting down persistent lock manager");
+            try {
+                persistentLockManager.shutdown();
+                log.info("Shutdown complete!");
+            } catch (Exception e) {
+                log.error("Exception while shutting down", e);
```

Comment:
Okay. I think we should probably inform people that a consequence of this could be that the backup lock might not have been released properly - and if there are problems with taking out the lock, to release it manually.

Also, would it be safe to log at `WARN`? I think if we have well established procedures for users to help themselves (and we do in the docs) then that should suffice.